curiouslearning/FeedTheMonsterJS
# Feat<FM-277-POC> Multi-Letter Selection Visual Effect

# Changes
- Added adjusted the size of stone handler when multi-letter selection is performed.

# How to test
Enable dev mode
- Option 1
1. In our FTM source code navigate to src/common/global-variables.ts.
2. Set the DebugMode from 'false' to 'true'.
3. Run the application with 'npm run dev' and in another cmd the command 'workbox injectManifest'.
4. Right click on the build/index.html and open it with LiveServer.
5. Game should run in developer mode with puzzle names indicated on level-selection.

- Option 2
1. On the start screen of the FTM app there is a gray button 'DEV'.
2. Press the 'DEV' button to enable the developer mode.
3. Game should run in developer mode with puzzle names indicated on level-selection.

After following the developer mode, do this steps:
- Press start and in level selection, select any "word" puzzle level (Follow dev mode to see the text indication of each levels).
- In the game play scene, you can now test this feature for multi-letter select.
- Follow the FM-276 for the acceptance criteria and full logic of the feature.
- Now it should show the stones size will increase as an indicator for multi-letter selection.

Ref: [<JIRA-CODE>](https://curiouslearning.atlassian.net/browse/FM-277)
